doc/tutorials/klondike/app/lib/main.dart

```diff
@@ -1,11 +1,12 @@
-import 'dart:html'; // ignore: avoid_web_libraries_in_flutter
+// import 'dart:html'; // ignore: avoid_web_libraries_in_flutter
 import 'package:flutter/widgets.dart';
 import 'step2/main.dart' as step2;
 import 'step3/main.dart' as step3;
 import 'step4/main.dart' as step4;
 
 void main() {
-  var page = window.location.search ?? '';
+  // var page = window.location.search ?? '';
+  var page = 'step4';
```

Comment:
This change should probably also be reverted

Reply:
It was accidentally committed and pushed. I have written to Spydon above re how to get the offending commit expunged from both repositories (my local and remote) and its history deleted. The changes are just my hacks to avoid compilation errors on Apple MacOS and get the Klondike app to run.

Reply:
The offending commit https://github.com/flame-engine/flame/commit/18f71bece67fad79b71b9185dc18101f81f0df27 has been reverted.